charts/loghouse/values.yaml

```diff
@@ -77,6 +77,8 @@ clickhouse:
   #node_selector:
   #  key: 'node-role/logging'
   #  value: 'clickhouse'
+  # Setup memory limit for clickhouse
+  memlimits: 7000
```

Comment:
@gyrter, it would be much better if we name it `memory_usage_megabytes`.
1. First of all, the word "limit" usually corresponds to Kubernetes limits and creates some ambiguity.
2. Secondly, all other parameters are written in full form, not shortened. And `_` is used as the word separator. It would be better to follow this convention.
3. Finally, there is a problem with the measurement units. In your implementation user have to guess what the unit is. So I propose in this case to add the name of the measure unit right to the parameter. Of course, it would be better to support suffixes widely used in Kubernetes (Gi, Mi, etc), but there are no easy ways of doing this in Helm.